report_qweb_pdf_watermark/models/report.py

```diff
@@ -115,11 +115,11 @@ def _run_wkhtmltopdf(
             except Exception as e:
                 logger.exception("Failed to load watermark", e)
 
-        if not pdf_watermark:
-            logger.error("No usable watermark found, got %s...", watermark[:100])
+        if not self.pdf_has_usable_pages(pdf_watermark):
             return result
 
-        if not self.pdf_has_usable_pages(pdf_watermark.numPages):
+        if not pdf_watermark:
+            logger.error("No usable watermark found, got %s...", watermark[:100])
```

Comment:
You've swapped the two checks around, but shouldn't the check for `if not pdf_watermark` come first? The other check actually goes into pdf_watermark.numPages, so seems like it should come after.